src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -196,12 +196,11 @@ const View& RenderTarget::getDefaultView() const
 ////////////////////////////////////////////////////////////
 IntRect RenderTarget::getViewport(const View& view) const
 {
-    float            width    = static_cast<float>(getSize().x);
-    float            height   = static_cast<float>(getSize().y);
-    const FloatRect& viewport = view.getViewport();
+    Vector2f  size(getSize());
+    Vector2f  offset(0.5f, 0.5f);
+    FloatRect viewport(view.getViewport());
```

Comment:
```suggestion
    const FloatRect& viewport = view.getViewport();
```
No need to change this line. SFML convention is to not use `()` to copy construct stack variables. Your personal style appears to be using `()` ubiquitously, but please just use the style that is already in use.

Reply:
OK